Source/Scene/getMagic.js

```diff
@@ -1,19 +0,0 @@
-/*global define*/
-define([
-        '../Core/defaultValue',
-        '../Core/getStringFromTypedArray'
-    ], function(
-        defaultValue,
-        getStringFromTypedArray) {
-    "use strict";
-
-    /**
-     * @private
-     */
-    var getMagic = function(uint8Array, byteOffset) {
```

Comment:
It was moved to `Source/Core/getMagic.js`

Also `getBasePath` was renamed to `getBaseUri`